app/assets/javascripts/cable/subscriptions/notifications.js

```diff
@@ -0,0 +1,12 @@
+App.cable.subscriptions.create('NotificationsChannel', {
+  received: function(data) {
+    if(data.new_matches_count == 1) {
+      $('.modal .new-matches-count').html('<h3>You have a new match!</h3>');
```

Comment:
Move the set of the jquery objects to outside, in the condition decide which test will be set